scripts/arbitrum_earn.py

```diff
@@ -12,11 +12,10 @@
 logger = logging.getLogger(__name__)
 
 
-def safe_earn(earner, vault, strategy):
+def safe_earn(earner: Earner, vault: contract, strategy: contract, name: str):
     try:
-        sett_name = strategy.functions.getName().call()
-        logger.info(f"+-----Earning {sett_name}-----+")
-        earner.earn(vault, strategy, sett_name=sett_name)
+        logger.info(f"+-----Earning {name}-----+")
+        earner.earn(vault, strategy, sett_name=name)
```

Comment:
Is it a sett_name if we pass there strat name?

Reply:
doesn't matter it's for logging, so will be the name field of metadata from the vault registration in registry